client/src/manifest.json

```diff
@@ -1,21 +1,35 @@
 {
-  "name": "navi",
   "short_name": "navi",
-  "start_url": "/",
+  "name": "Navi - a navigator app for the Udacity Grow with Google project",
+  "start_url": "/?utm_source=homescreen",
   "display": "standalone",
-  "orientation": "portrait",
-  "background_color": "#fff",
-  "theme_color": "#673ab8",
-  "icons": [
-    {
-      "src": "/assets/icons/android-chrome-192x192.png",
-      "type": "image/png",
-      "sizes": "192x192"
-    },
-    {
-      "src": "/assets/icons/android-chrome-512x512.png",
+  "orientation": "landscape",
```

Comment:
I think we should stick with portrait here for now and consider if we can use  
[Device Orientation API](https://developer.mozilla.org/en-US/docs/WebAPI/Detecting_device_orientation)
or [Screen Orientation API](https://developer.mozilla.org/en-US/docs/WebAPI/Managing_screen_orientation)
later on as an enhancement